microsoft-edge/developer/index.yml

```diff
@@ -100,20 +100,26 @@ landingContent:
     linkLists:
```

Comment:
@jasonstephen15 - Are these the right articles to spotlight in the card of the landing page & of the hubpage?
[Landing page > WebView2 card](https://review.docs.microsoft.com/en-us/microsoft-edge/developer/?branch=pr-en-us-1690)
[Hub page > WebView2 card](https://review.docs.microsoft.com/en-us/microsoft-edge/?branch=pr-en-us-1690#microsoft-edge-for-developers)

Limits for a card:
* 6 article-types sections in a card, max.
* 10 article-links in an article-type section in a card, max.
eg a card can have up to 6 sections, w/ 10 article-links each.  Compare the height of other cards, try to be similar.